src/Microsoft.Build.Sql/Microsoft.Build.Sql.csproj

```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SqlServer.DacFx" Version="$(DacFxPackageVersion)" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.1.1" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.1.3" GeneratePathProperty="true" />
```

Comment:
Why not 5.1.4?

Reply:
I don't think there's any risk of using 5.1.4 but we're just keeping the dependencies consistent with DacFx NuGet